library/tasks/ModifyGtf.wdl

```diff
@@ -0,0 +1,51 @@
+task ReplaceGeneNameWithGeneID {
```

Comment:
Great changes. Can you comment on why we don't we take this one step further?

1. execute this script one time
2. save it somewhere
3. pass in the modified GTF file to the Optimus script

Reply:
Thank you for your feedback, Ambrose! I think while it seems nice to not have to re-modify the gtf each time we run the workflow, I think there are some things that make it sub-optimal to preemptively create & save the file:

1. That would involve a change to our infrastructure rather than a workflow, and this approach isn't set in stone. We are still testing whether this is the best way to modify the gtf, and if the 3rd party tools we create this for get changed/updated we may no longer need this file.

2. From what I understand from discussions with Comp Bios, we don't want the modified gtf to be an output or otherwise accessible since it's technically a workaround and a big no-no.

3. The gtf is very small (comparatively) and the operation to modify it fast enough that it isn't really an issue to do. Only 2 tasks need the output from ReplaceGeneNameWithGeneID, so hopefully there is time for it to run in parallel with other tasks. Furthermore, if call-cacheing is enabled, it will immediately return a cache hit.